docs/clearml_agent/clearml_agent_deployment_slurm.md

```diff
@@ -89,13 +89,52 @@ template specification attached to the queue.
 ## Slurm with Singularity
 If you are running Slurm with Singularity containers support, set the following:
 
-1. Make sure your `sbatch` template contains:
+1. Create a batch template. Make sure your `sbatch` template contains:
```

Comment:
@pollfly Please note that exec arguments are supported (see example you removed)